src/qt/walletmodel.cpp

```diff
@@ -28,13 +28,25 @@
 #include <wallet/coincontrol.h>
 #include <wallet/wallet.h> // for CRecipient
 
+// namecoin API-related includes
+// TODO: figure out which of these includes are actually still necessary for name_list
+#include <names/common.h>
+#include <names/main.h>
+#include <qt/nametablemodel.h>
+#include <rpc/client.h>
+#include <rpc/server.h>
+
 #include <stdint.h>
 
 #include <QDebug>
 #include <QMessageBox>
 #include <QSet>
 #include <QTimer>
 
+// TODO: figure out which of these includes are actually still necessary for name_list
+#include <boost/foreach.hpp>
```

Comment:
At least the boost include should not be necessary I think.  The codebase has stopped using boost pretty much everywhere for a long time, and we should definitely use C++11 for-each loops instead of boost.

Reply:
You are correct, the boost include wasn't needed anymore.